### ISSUE TITLE
need for Docker image

### ISSUE THREAD

**Issue by pengbo233** [YosefLab/Compass#111]
Thank you very much for providing this tool. Can you provide a Docker image?

**Comment by rajuadduri** [comment]
Yes. I could not create the docker image of compass. If the developers push a docker image, it will help me too.

**Comment by voges** [comment]
Hello everyone! We are also working with Compass and thought that a Docker setup would be nice...

Here it is: https://github.com/voges/compass-docker/tree/main

The only remaining obstacle is that you are legally required to obtain your own copy of the Cplex Installer from IBM, but we provide detailed instructions for that.

Please leave a star if you like it. :)

**Comment by rajuadduri** [comment]
Thank you Voges.
